server/server.js

```diff
@@ -0,0 +1,41 @@
+/* eslint no-console:0 */
+var loopback = require('loopback');
+var boot = require('loopback-boot');
+var ServiceManager = require('./service-manager');
+
+function server(serviceManager) {
```

Comment:
@rmg has the right idea. This is a factory which returns an LB app, not a constructor.